### PR TITLE
SAK-31468 rename 'gradebook' to 'gradebook classic' and 'gradebookng'…

### PR DESCRIPTION
**Pull request by steveswinsburg**
… to 'gradebook'.

### CODE REVIEW

**Comment by juanjmerono** [comment]
Why is this change in both conversion scripts 11 and 12? I think it should be only in 11 conversion script.

**Comment by steveswinsburg** [comment]
Becuase GradebookNG is currently in 12 and this fix is targetted for 11 as well. There is no way to 'merge' back to the 11 conversion scripts as its a completely separate file so it's done here.

**Comment by buckett** [comment]
On 6 July 2016 at 09:40, Steve Swinsburg <notifications@github.com> wrote:

> Becuase GradebookNG is currently in 12 and this fix is targetted for 11 as
> well. There is no way to 'merge' back to the 11 conversion scripts as its a
> completely separate file so it's done here.
>

But isn't it the case that anyone running upgrading to 12 will have already
run the upgrade script for 11?

-- 
  Matthew Buckett

**Comment by steveswinsburg** [comment]
That is in fact, true. 🍰

File: config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/tool/tools.properties [hunk 1]
```diff
@@ -249,3 +249,5 @@ sakai.delegatedaccess.shopping.description=Search for available sites for shoppi
 sakai.adminsiteperms.title=Admin Site Perms
 sakai.adminsiteperms.description=Allows adminstrators to control permissions for all sites of a type and roles within that site
 
+sakai.gradebookng = Gradebook
```
Comment (danielmerino) [inline config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/tool/tools.properties]: This key will not work, will it? AFAIK every tool needs to have a ".title" and ".description" endings in their keys.

We have changed it in our local 11.4 to have GradebookNG correctly internationalized.

**Pull request merged**